app/src/organisms/SendProtocolToOT3Slideout/__tests__/SendProtocolToOT3Slideout.test.tsx

```diff
@@ -246,7 +246,7 @@ describe('SendProtocolToOT3Slideout', () => {
       onCloseClick: jest.fn(),
       isExpanded: true,
     })
-    const sendButton = getByRole('button', { name: 'Send' })
+    const sendButton = getByRole('button', { name: 'Proceed to setup' })
```

Comment:
The name is not Send so changing would be better.

```suggestion
    const proceedButton = getByRole('button', { name: 'Proceed to setup' })
```